lisp/ess-custom.el

```diff
@@ -2108,27 +2113,26 @@ for help files.  The default value is nil for other systems."
 
 ;;;;; names for communication using MS-Windows 9x/NT ddeclient mechanism
 
-(defcustom inferior-ess-ddeclient nil
+(defcustom inferior-ess-ddeclient "Initial"
```

Comment:
i will look at this later in the week.  my prior is that dde should be retained as is, with no changes, so that S-Plus gui will continue to work on windows.
similarly ess-sp6w.el should be retained.

when i invented the term "Initial" i had areason.  please don't change it until i recall the situation.

Sent from my iPhone

> On Apr 3, 2016, at 09:25, Lionel Henry <notifications@github.com> wrote:
> 
> In lisp/ess-custom.el:
> 
> > @@ -2108,27 +2113,26 @@ for help files.  The default value is nil for other systems."
> >  
> >  ;;;;; names for communication using MS-Windows 9x/NT ddeclient mechanism
> >  
> > -(defcustom inferior-ess-ddeclient nil
> > +(defcustom inferior-ess-ddeclient "Initial"
> Why "Initial". Looks like a wart.
> 
> yes, someone initialised that variable to "Initial" in ess-site.el... I should have changed the code to deal with nil rather than "Initial" but I think I was starting to give up at that point ;) I'll change it to nil.
> 
> I think we can drop support for DDE. I doubt anyone uses that anymore.
> 
> It's needed for S-plus on Windows. I'm not against dropping support, but let's see if @rmheiberger manages to run the latest ESS with these environments.
> 
> No-one tests is anyways.
> 
> Yes, we really need basic tests for all the inferiors officially supported, and some people on the team in charge of running them before each release. I think we should drop support for all the systems we are not able to test.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>